src/valid/analyzer.rs

```diff
@@ -16,15 +16,19 @@ use std::ops;
 
 pub type NonUniformResult = Option<Handle<crate::Expression>>;
 
+// Remove this once we update our uniformity analysis and
+// add support for the `derivative_uniformity` diagnostic
+const DISABLE_UNIFORMITY_REQ_FOR_FRAGMENT_STAGE: bool = true;
```

Comment:
This seems like more trouble than it's worth. But it's temporary, so let's go with it.

Reply:
Yeah, I wasn't sure how much of the uniformity analysis code was going to change to get us up to spec, so didn't want to spend too much time on this.